### DISCUSSION TITLE
en/chapter_array_and_linkedlist/array/

### DISCUSSION THREAD

**Discussion by giscus** [krahets/hello-algo#1553] [Announcements]
# en/chapter_array_and_linkedlist/array/

Data Structures and Algorithms Crash Course with Animated Illustrations and Off-the-Shelf Code

https://www.hello-algo.com/en/chapter_array_and_linkedlist/array/

**Comment by ghost** [comment]
I was hoping to be able to understand linked lists and stacks and queues and the other stuff, but I think I'll just get to read about the theory going through this. I wish you guys had 10-20 examples per concepts of an array, list, linked list, etc.
